### ISSUE TITLE
Jump to def, back from def, jump to def with split views.

### ISSUE THREAD

**Issue by bill-mybiz** [msaraiva/atom-elixir#39]
Ubuntu 16.04, Atom 1.8.0

Steps to repro:
1. Open two window panes with the two different pertinent source files: One where the function is declared, and one where it is consumed.
2. In the file where the function is consumed, hit `alt+down` to go to the definition. This should take you over to the other pane.
3. Now hit `alt+up` to return to where you just were. This returns to the correct location on the initial pane.
4. Now hit `alt+down` again (as in, "what was that I was just looking at again?"). For me, when I navigate to the definition the second time, it splits the source file between the two panes. 

In the correct pane, the tab still exists. But the source file is now on the left pane and things are pretty wonky at this point:

![mixed_panes_jump_definition](https://cloud.githubusercontent.com/assets/1253349/16898218/df75016a-4b93-11e6-9f44-d8d249b24249.png)

Does anyone else have this problem?

Btw, this library is pretty freakin awesome. I :heart: me some autocomplete and jump-to-def! :+1: (_any chance for refactoring in the future?_ :wink: )

**Comment by msaraiva** [comment]
> Does anyone else have this problem?

Yes. I was able to reproduce the problem on OSX. I'll take a look at it.

> any chance for refactoring in the future?

Well, it is planned, but I'll need to find some time to do it. I'm not sure if I'll be able to implement this in the next couple of months. Let's see how it goes.

Thanks for reporting this issue!
